### PR TITLE
docs: explain how to combine plugins

### PR DESCRIPTION
**Pull request by tchollingsworth**
[I was confused about how to do this on the forum until @schoen set me straight](https://community.letsencrypt.org/t/urn-acme-error-tls-the-server-experienced-a-tls-error-during-domain-verification-remote-error-tls-handshake-failure/36770/10), so I documented it with an example that solves a common usecase.

### CODE REVIEW

**Comment by bmw** [comment]
@SwartzCr, would you like to review this or should I?

File: docs/using.rst [hunk 1]
```diff
@@ -205,6 +205,26 @@ perform the authentication procedure and/or clean up after it by using
...
+Sometimes you may want to specify a combination of distinct authenticator and
+installer plugins. To do so, specify the authenticator plugin with
+``--authenticator`` or ``-a`` and the installer plugin with ``--installer`` or
+``-i``.
+
+For instance, you may want to create a certificate using the webroot_ plugin
+for authentication and the apache_ plugin for installation, perhaps because you
+use a proxy or CDN for SSL and only want to secure the connection between them
+and your origin server, which cannot use the tls-sni-01_ challenge due to the
+intermediate proxy.)
```
Comment (SwartzCr) [inline docs/using.rst]: I don't see an open parenthesis here, maybe this closing one should be removed

**Comment by SwartzCr** [comment]
Looks good to me, besides one minor nit. Fix it and I'm happy to merge! Thanks for making this

File: docs/using.rst (hunk 1 above)
Comment (tchollingsworth) [inline docs/using.rst]: Sorry about that. I tried very hard to make this two sentences but everything I came up with was more annoying and repetitive th

**Comment by SwartzCr** [comment]
I am the worst human for letting this sit so long - sorry about that, merging it now!

**Pull request merged**